src/app/content/highlights/components/ShowMyHighlights.tsx

```diff
@@ -77,6 +77,7 @@ class ShowMyHighlights extends Component<ShowMyHighlightsProps, { showGoToTop: b
       <Styled.ShowMyHighlightsBody
         ref={this.myHighlightsBodyRef}
         data-testid='show-myhighlights-body'
+        data-analytics-region='highlighting-my-highlights-popup'
```

Comment:
lets try to keep this short
```suggestion
        data-analytics-region='MH popup'
```